js/src/common/Translator.ts

```diff
@@ -45,7 +55,7 @@ export default class Translator {
     return parameters;
   }
 
-  trans(id, parameters) {
+  trans(id: string, parameters: any) {
```

Comment:
Could we replace `any` with something else? Should `parameters` also be marked as optional using `?:`?

Maybe `parameters?: Record<string, unknown>` would work?

Reply:
That makes sense to me. Although then we should default to an empty object.